.gitignore

```diff
@@ -5,4 +5,6 @@ Internxt Drive.app
 build
 *.p8
 app_store_key.json
-report.xml
\ No newline at end of file
+report.xml
+*.xcuserstate
+InternxtDesktop.xcodeproj/xcuserdata/richardascanio.xcuserdatad/xcschemes/xcschememanagement.plist
```

Comment:
Try this: `xcuserdata/*` or `xcuserdata/**/*`, don't hardcode your own name here